include/amici/sundials_matrix_wrapper.h

```diff
@@ -252,6 +252,28 @@ class SUNMatrixWrapper {
                          sunindextype *w, gsl::span<realtype> x,
                          const sunindextype mark,
                          SUNMatrixWrapper *C, sunindextype nnz) const;
+    
+    /**
+     * @brief Computes transpose A' of sparse matrix A and writes is to the matrix C = alpha * A'
+     *
+     * @param C output matrix (sparse or dense)
+     * @param alpha scalar multiplier
+     */
+    void transpose(SUNMatrix C, const realtype alpha) const;
```

Comment:
Don't we want to use `SUNMatrixWrapper`?

Reply:
We use this to directly write to the raw SUNMatrix that comes from sundials, so this needs to be SUNMatrix. Could add additional overload for `SUNMatrixWrapper`, but would currently be unused.